typings/committee-contract.d.ts

```diff
@@ -6,12 +6,14 @@ export interface CommitteeChangedEvent {
     addrs: string[];
     orbsAddrs: string[];
     weights: (number | BN)[];
+    ips: string[];
```

Comment:
Do you see IPs in these events? I don't recall us ever including this in committee events..